### PR TITLE
AKO 1.10.3 documents changes for Aviinfrasetting

### PR DESCRIPTION
**Pull request by akshayhavile**

### CODE REVIEW

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

**Comment by akshayhavile** [comment]
skip jira-id-check

**Comment by akshayhavile** [comment]
build ako

**Comment by akshayhavile** [comment]
build ako

File: docs/crds/avinfrasetting.md [hunk 1]
```diff
@@ -120,22 +120,49 @@ Please make sure that the SEGs have no member Service Engines deployed, before s
 
 AviInfraSetting CRD can be used to configure VIP networks for virtualservices created corresponding to Services/Ingresses/Openshift Routes. The Networks must be present in the Avi Controller prior to this CRD creation.
 
+Wth AKO 1.10.3, `networkUUID` field has been introduced as part of `vipNetworks` setting to choose appropriate network. User can specify either `networkname` or `networkUUID` while specifying the network.
```
Comment (swathinsankaran) [inline docs/crds/avinfrasetting.md:123]: `networkName` instead of `networkname` here.

File: docs/crds/avinfrasetting.md [hunk 2]
```diff
@@ -120,22 +120,49 @@ Please make sure that the SEGs have no member Service Engines deployed, before s
...
+***Note***
+If there are duplicate network entries at Avi-Controller side, AKO recommends to use `networkUUID` while choosing `vipNetwork` for new aviinfrasetting object creation.
+
 Note that multiple networks names can be added to the CRD (only in case of AWS cloud). The Avi virtualservices will acquire a VIP from each of these specified networks. Failure in allocating even a single vip (for example, in case of IP exhaustion) **will** result in complete failure of entire request. *This is same as vip allocation failures in single vip.*
 
 #### Configure Pool Placement Networks
 
 AviInfraSetting CRD can be used to configure Pool Placement Settings on the AKO created Pools in order for the Service Engines to place the Pools on appropriate interfaces.
 
+Wth AKO 1.10.3, `networkUUID` field has been introduced as part of `nodeNetworks` setting to choose appropriate network. User can specify either `networkname` or `networkUUID` while specifying the network.
```
Comment (swathinsankaran) [inline docs/crds/avinfrasetting.md:147]: `networkName` instead of `networkname`.

File: docs/faq.md [hunk 3]
```diff
@@ -291,3 +291,16 @@ Currently, AKO beyond two has not been tested. Hence we don't claim the support.
 #### What happens during a Split brain scenario in HA?
 
 AKO detects this during the periodic refresh of the lease lock object and makes itself a passive AKO.
+
+#### With AKO 1.10.3, field `networkUUID` has been added as part of `vipNetworkList` and `nodeNetworkList`. Does it impact existing AviInfra objects?
+
+When customer upgrade from older AKO version to AKO 1.10.3 or later, customer has to always upgrade CRD schemas as part of upgrade process.
+But there is no impact on existing AviInfraSetting objects and user doesn't require to do changes to existing objects. AKO accepts either `networkName` or `networkUUID` as part of these fields.  If there are duplicates network at NSX Advance Loadbalancer side, AKO recommends the use of `networkUUID` while creating new AviInfrasetting objects.
```
Comment (swathinsankaran) [inline docs/faq.md:298]: `NSX Advanced LoadBalancer` instead of `NSX Advance Loadbalancer` here.

File: docs/faq.md [hunk 4]
```diff
@@ -291,3 +291,16 @@ Currently, AKO beyond two has not been tested. Hence we don't claim the support.
...
+
+#### How can I find out `networkUUID` for the `networkName`?
+
+One of the way to figure out `networkUUID` for given `networkName` is as follows:
```
Comment (swathinsankaran) [inline docs/faq.md:302]: Should we list down all the ways or do we have to reword this if this is the only way?

File: docs/faq.md [hunk 5]
```diff
@@ -291,3 +291,16 @@ Currently, AKO beyond two has not been tested. Hence we don't claim the support.
...
+1. Login to NSX Advanced Loadbalancer on any web browser.
+2. On Web browser, type fqdn as `https://<controller-ip-or-fqdn>/api/network?name=<NameOfNetwork>`
+3. Choose `uuid` field of one of the retrieved records.
+4. In case of multiple records are retrieved, user has to use `switch name` and/or `configured subnet` fields of the record, to choose correct network.
```
Comment (swathinsankaran) [inline docs/faq.md:306]: we have to provide the correct field name here like `switch_name` and `configured_subnet`.

**Review by swathinsankaran — COMMENTED** [review]
overall the document looks good to me.
I have provided some minor feedback.

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

**Comment by akshayhavile** [comment]
skip jira-id-check

**Review by swathinsankaran — APPROVED** [review]
LGTM

**Comment by swathinsankaran** [comment]
build ako

**Pull request merged**